nodes/core/io/32-udp.js

```diff
@@ -126,7 +126,7 @@ module.exports = function(RED) {
 
         var sock;
         if (udpInputPortsInUse[this.outport]) {
-            sock = udpInputPortsInUse[this.outport];
+            sock = udpInputPortsInUse[this.outport || this.port];
```

Comment:
If `this.outport` isn't defined then we won't be in this `if` block for this change to have any affect. The `if` statement on line 128 need to change to match.

Reply:
good catch - thanks